.github/MISSING_ISSUE_TEMPLATE_AUTOREPLY.md

```diff
@@ -0,0 +1,4 @@
+Hey @{{ sender.login }}, thanks for opening an issue!
+
+Unfortunately, it seems you are missing information from the [issue template](/.github/ISSUE_TEMPLATE.md).
+Please provide all nexxessary information as described in our [contribution guidelines](/.github/CONTRIBUTING.md).
```

Comment:
"necessary"
Also, some issues are not issues (support requests, feature requests, etc, or very simplistic but accurate bug reports) so maybe reword the message with something like "make sure it has all the necessary information as described in ..."